e2e_samples/temperature_events/Infra/setup.sh

```diff
@@ -48,3 +48,19 @@ az keyvault secret set --name tfstate-storage-key-dev --value $ACCOUNT_KEY_DEV -
 # Display information
 echo "Run the following command to initialize Terraform to store its state into Azure Storage:"
 echo "terraform init -backend-config=\"storage_account_name=$TF_STATE_STORAGE_ACCOUNT_NAME\"dev -backend-config=\"container_name=$TF_STATE_CONTAINER_NAME\" -backend-config=\"access_key=\$(az keyvault secret show --name tfstate-storage-key --vault-name $KEYVAULT_NAME --query value -o tsv)\" -backend-config=\"key=terraform.tfstate\""
+
+# Create Service Principal
```

Comment:
This is cool

Reply:
Fixed minor issues we found during our testing.